### DISCUSSION TITLE
model-viewer don't work in Firefox - TypeError: NetworkError when attempting to fetch resource.

### DISCUSSION THREAD

**Discussion by Fabio-Steinberger** [google/model-viewer#3570] [Q&A]
Hello, everyone,
I try to load a glb file with the model viewer, but I always get this error message in the console: 

**"TypeError: NetworkError when attempting to fetch resource."** 

However, this message only appears in Firefox, all other browsers work. My model looks like this:

"<model-viewer alt="" src="plugins/modelviewer_jtl/glb/E280608.N.glb" ar="" ar-modes="webxr scene-viewer quick-look" camera-controls="" enable-pan ="" id="mv" camera-orbit="0deg 75deg auto" exposure="2" shadow-intensity="1.48" shadow-softness="0.4" interaction-prompt-threshold="500" data-js-focus -visible="" ar-status="not-presenting"></model-viewer>"

Unfortunately I didn't find anything about it. Thanks in advance

**Comment by elalish** [comment]
Best guess is some kind of server configuration issue (CORS is a popular error, though that will usually show up in the message). Maybe it doesn't like `.N.glb`? Maybe check the Firefox forums.
